Integrations/integration-AWS-IAM.yml

```diff
@@ -1024,6 +1024,75 @@ script:
         except Exception as e:
             return raise_error(e)
 
+
+    def get_account_password_policy(args):
+        try:
+            client = aws_session(
+                    roleArn = args.get('roleArn'),
+                    roleSessionName = args.get('roleSessionName'),
+                    roleSessionDuration = args.get('roleSessionDuration'),
+                )
+
+            response = client.get_account_password_policy()
+
+            if 'PasswordPolicy' not in response:
+                raise_error('Password policy does not exists')
```

Comment:
I've removed it from the code. I meant to write it using `except client.exceptions.NoSuchEntityException as e:` like in `update_account_password_policy`, so that it's consistent with the rest, when IAM resource is queried but the resource doesn't exists, the exception will be caught by `return raise_error(e)`.